ProjectSidewalk/SidewalkWebpage
# 3249 add speed limit indicator to explore page with OSM data

Resolves #3249

Using OpenStreetMap data through the Overpass API, this PR adds a speed limit indicator to the UI of the explore page.

##### Before/After screenshots (if applicable)

Before:

![image](https://github.com/user-attachments/assets/cbee39eb-0645-4fde-ba69-eab584c296c0)

After:

![image](https://github.com/user-attachments/assets/5e6cd2e5-bcf8-45fa-8895-9de57475381a)

##### Testing instructions
1. Go to the explore page and you should see the speed limit indicator provided the OSM data exists for your current location
2. Try moving around a bit in Street View and see how the speed limit changes (this might take a while as the speed limit can be pretty consistent)
3. You can see the requests in the network tab of devtools

##### Things to check before submitting the PR
